azure-pipelines.yml

```diff
@@ -209,26 +209,42 @@ stages:
     jobs:
       - template: /eng/common/templates/job/job.yml
         parameters:
-          name: Validate_Arcade_with_Roslyn
-          displayName: Validate Arcade with Roslyn
-          timeoutInMinutes: 210
+          name: Validate_Arcade_With_Consumer_Repositories
+          displayName: Validate Arcade with Consumer Repositories
+          timeoutInMinutes: 240
           variables:
             - group: Publish-Build-Assets
             - group: DotNetBot-GitHub
             - group: DotNet-Blob-Feed
             - group: DotNet-Symbol-Server-Pats
             - group: DotNet-VSTS-Infra-Access
+          strategy:
+            matrix:
+              ValidateWithRuntime:
+                _azdoOrg: "dnceng"
+                _azdoProject: "internal"
+                _buildDefinitionId: 679
+                _githubRepoName: "runtime"
```

Comment:
Since roslyn is the exception in that it build from GitHub, would it make sense to make the github repository the optional parameter? I don't think we care about the github repo at all when it comes to runtime, or any of the other repos that we're going to validate against.

Reply:
Ah, I see that you're cloning the github repo always. I kind of expected this to clone the azure devops mirror instead since that is the actual repo where the official build runs from. That way you don't have to add a remote, since you're cloning the real repo. This approach seems fine to me as well, if a little roundabout.

There's a very very very small concern where the azdo branch might have different commits than github, and the mirror would be broken in that case, so we'd end up building against the wrong branch, but the chances of that are so small, and all sorts of things would be broken anyways that I think this is fine as well.